mailpoet/lib/Entities/SegmentEntity.php

```diff
@@ -67,6 +67,12 @@ class SegmentEntity {
    */
   private $averageEngagementScoreUpdatedAt;
 
+  /**
+   * @ORM\Column(type="integer", nullable=true, options={"default" : 0})
+   * @var int|null
+   */
+  private $displayInManageSubscriptionPage;
+
```

Comment:
Hmm, this is a boolean flag, correct? The doctrine has a boolean type that should be used for such stuff. Please see an [example usage in SubscriberEntity](https://github.com/mailpoet/mailpoet/blob/2d835cdec1df34608f8043f0df7125cbe9e0e93c/mailpoet/lib/Entities/SubscriberEntity.php#L53-L57).
I'm sorry I didn't notice that in the initial review. Could you please change the type to boolean here?
If we expect to support more values then true/false, then please ignore this comment.

Reply:
Thanks for the review @costasovo 
It's 1 or 0 (integer). 
We could typecast it to boolean as the column shouldn’t hold any other value.